pkg/config/config.go

```diff
@@ -13,26 +13,29 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
-// These constants MUST be equal to the corresponding names in service definition in definitions/install.yaml
+// These MUST be equal to the corresponding names in service definition in definitions/install.yaml
 
 // webhook configuration names
-const (
-	// PolicyValidatingWebhookConfigurationName default policy validating webhook configuration name
+var (
+	WebhookConfigurationsNamePrefix = osutils.GetEnvWithFallback("KYVERNO_WEBHOOK_CONFIGS_NAME_PREFIX", "kyverno")
+
+	// PolicyValidatingWebhookConfigurationName policy VALIDATING webhook configuration name
 	PolicyValidatingWebhookConfigurationName = "kyverno-policy-validating-webhook-cfg"
 	// ValidatingWebhookConfigurationName ...
 	ValidatingWebhookConfigurationName = "kyverno-resource-validating-webhook-cfg"
 	// ExceptionValidatingWebhookConfigurationName ...
 	ExceptionValidatingWebhookConfigurationName = "kyverno-exception-validating-webhook-cfg"
 	// CleanupValidatingWebhookConfigurationName ...
 	CleanupValidatingWebhookConfigurationName = "kyverno-cleanup-validating-webhook-cfg"
-	// PolicyMutatingWebhookConfigurationName default policy mutating webhook configuration name
-	PolicyMutatingWebhookConfigurationName = "kyverno-policy-mutating-webhook-cfg"
-	// MutatingWebhookConfigurationName default resource mutating webhook configuration name
-	MutatingWebhookConfigurationName = "kyverno-resource-mutating-webhook-cfg"
-	// VerifyMutatingWebhookConfigurationName default verify mutating webhook configuration name
-	VerifyMutatingWebhookConfigurationName = "kyverno-verify-mutating-webhook-cfg"
-	// TtlValidatingWebhookConfigurationName ttl label validating webhook configuration name
-	TtlValidatingWebhookConfigurationName = "kyverno-ttl-validating-webhook-cfg"
+
+	// PolicyMutatingWebhookConfigurationName policy MUTATING webhook configuration name
+	PolicyMutatingWebhookConfigurationName = fmt.Sprintf("%s-policy-mutating-webhook-cfg", WebhookConfigurationsNamePrefix)
```

Comment:
Why we need to add a prefix to policy mutation webhook?

Since there's no way to have other mutation webhooks that matches kyverno policies, then it doesn't make sense to add prefix for them. Correct me if I misunderstood the use case.

Reply:
Ya, there is already a conversation around this in the issue https://github.com/kyverno/kyverno/issues/7997#issuecomment-1672349293. Functionally speaking, it sounds like `resource-mutating-webhook` is the only one that makes sense to rename.

However, I was trying to be consistent with naming such that if you `kubectl get mutatingwebhookconfigurations` all the Kyverno ones are contiguous. Does that make sense?